pkg/rego/metadata.go

```diff
@@ -213,6 +213,19 @@ func (m *MetadataRetriever) updateMetadata(meta map[string]interface{}, metadata
 			metadata.Frameworks[framework.Framework(fw)] = sections
 		}
 	}
+	if raw, ok := meta["related_resources"]; ok {
```

Comment:
I have a feeling this change will increase the cyclomatic compexity :thinking:

Reply:
oh no